src/main/java/com/google/sps/servlets/ContestsServlet.java

```diff
@@ -66,22 +67,36 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     Election election = Election.fromEntity(electionEntityOptional.get());
 
     List<JsonElement> contestJsonList =
-        ImmutableSet.copyOf(election.getContests())
+        election
+            .getContests()
             .stream()
             .map(id -> KeyFactory.createKey(Contest.ENTITY_KIND, id.longValue()))
             .map(key -> ServletUtils.getFromDatastore(datastore, key))
-            .map(entity -> Contest.fromEntity(entity).toJsonString(datastore))
-            .map(jsonString -> JsonParser.parseString(jsonString))
-            .collect(ImmutableList.toImmutableList());
+            .map(
+                entity -> {
+                  return entity.isPresent()
```

Comment:
No need for "return" just remove the {}. 

.map(entity -> entity.isPresent() ? asd : basdf)

Reply:
Oh, good to know! will do.